tools/sz_repo_cli/lib/src/common/src/app_store_connect_utils.dart

```diff
@@ -135,6 +155,15 @@ Future<int> _getLatestBuildNumberFromAppStoreConnect({
   }
 }
 
+// Sets the required permissions for deploying Flutter macOS.
+//
+// Workaround for https://github.com/flutter/flutter/issues/132725. Can be
+// removed with Flutter v3.13.5.
```

Comment:
The fix for https://github.com/flutter/flutter/issues/132725 will be included in the next Flutter hot fix update (v3.13.5) 👍